.gitignore

```diff
@@ -14,3 +14,15 @@
 # Dependency directories (remove the comment below to include it)
 # vendor/
 dist/
+
+# OS related files
+.DS_Store
+
+# Editor/IDE files
+.idea
+
+# SQLite Database
+db/dblab.db
+
```

Comment:
Ok, but we need a way to delete an existing file called `db/dblab.db` and recreate it for testing purposes.

Reply:
Do you think it could be works if I deleted first the existing `db/dblab.db` file, and then I'll add it to `.gitignore` file

Reply:
Yeah, it works fine. I think we can merge as it is rn.